devmgmtV2/controllers/filemgmt.controller.js

```diff
@@ -289,7 +289,7 @@ let deleteFileFromDisk = (req, res) => {
   if (ext === '.ecar') {
     cmd = `rm -rf ${file}* ${json_dir}* ${xcontent}*`;
   } else {
-    if(!(dir.startsWith('/home/admin/diksha'))){
+    if((dir.startsWith('/home/admin')) && (!(dir.startsWith('/home/admin/diksha')))){
```

Comment:
can you please add a comment here, why are we doing this ?

Reply:
Comments added in commit : 85fa141b69f6fed2868d003eabf11492bd60fd6a.